brightchain-lib/src/lib/documents/base/document.ts

```
@@ -0,0 +1,130 @@
+   * Load document from storage
+   * To be implemented by subclasses
+   */
+  public async load(): Promise<TOperational> {
+    throw new NotImplementedError();
+  }
+
+  /**
+   * Delete document from storage
+   * To be implemented by subclasses
+   */
+  public async delete(operational: TOperational): Promise<void> {
```

ESLint

[error] 'operational' is defined but never used.

(@typescript-eslint/no-unused-vars)